docs/index.html

```diff
@@ -2,6 +2,7 @@
 <html>
     <head>
         <script src="bundle.js" type="text/javascript" charset="utf-8"></script>
+        <link rel="stylesheet" type="text/css" href="https://assets.clever.com/design/style_0.9.2.css">
```

Comment:
So does that mean updating the docs now has one more step of also the version here and uploading to the CDN?

I'd make those instructions visible somewhere at a minimum; or to set it up so that some static URL always has the latest styles; though truth be told I actually don't see much of a benefit of referring to the CDN for the docs page as this isn't as mission critical a service.